ui/lib/relayEnvironment.js

```diff
@@ -21,14 +21,14 @@ export let subscriptionClient;
 if (typeof window !== "undefined"){
   subscriptionClient = new SubscriptionClient("ws://"+window.location.host+"/api/system/graphql/query", {
     reconnect : true,
+    minTimeout: 4000
```

Comment:
This doesn't address root cause of the issue, does it? 

Why is the client choking?

Reply:
Yes, this might be the root cause. The connection closes itself even before it acknowledge the server response, take it as a relay bug or anything else, this has been the reason towards all the *subscription initialisation* bugs, now the operator and meshsync subscriptions updates and rest of them are also caught on fire with speed.